src/ucs/debug/memtrack.c

```diff
@@ -398,7 +398,7 @@ int ucs_posix_memalign_realloc(void **ptr, size_t boundary, size_t size,
     int ret;
 
 #ifndef ENABLE_MEMTRACK
-    old_size = malloc_usable_size();
+    old_size = malloc_usable_size(*ptr);
```

Comment:
i think we can use it in any case (even if memtrack is enabled) to simplify the code